bgpd/bgp_nexthop.c

```diff
@@ -967,39 +1302,81 @@ DEFUN (show_ip_bgp_nexthop,
 	    || argv_find(argv, argc, "X:X::X:X", &nh_idx))
 		nhop_ip = argv[nh_idx]->arg;
 
-	return show_ip_bgp_nexthop_table(vty, vrf, nhop_ip, false);
+	if (argv_find(argv, argc, "detail", &idx))
```

Comment:
Convert to DEFPY, and not needed this parsing.

Reply:
Fixed it